src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -59,7 +59,8 @@ private static ArgumentParser getArgumentParser() {
                 .metavar("LOCATION")
                 .help("The GitHub URL or disk locations to clone repository.");
 
-        mutexParser.addArgument("-view")
+        parser.addArgument("-view")
+                .nargs("*")
```

Comment:
Since the number of args is expected to be 0 or 1, also args >1 will be silently ignored, isn't `?` a better fit?
`.nargs("*")` -> `.nargs("?")`

Reply:
@yong24s I tried this, but it seems if we just called `-view` without any parameter, it will give a `null` value as well, which is the same as not indicating the `-view` argument.

Reply:
Try this:

```
parser.addArgument("-view")
      .setConst(Paths.get(""))
      .nargs("?")
      .metavar("PATH")
      .type(new ReportFolderArgumentType())      
      .help("Starts a server to display the dashboard in the provided directory.");
```

`.setConst(Paths.get(""))` be used when no value is given.

Reply:
@myteo 

Please implement the above comment.